crates/sui-rosetta/src/types.rs

```diff
@@ -899,6 +956,24 @@ impl InternalOperation {
                 builder.pay_sui(recipients, amounts)?;
```

Comment:
in order to not misinterpret a transaction as pay sui in rosetta, we might want to change this to include the coin type tags as well, maybe we can have a follow up PR once we have unblocked CB's integration?

lets add a TODO here for now.

Reply:
Not sure what exactly you have in mind here. Do you suggest to replace with `pay` instead? Like it happens for other coins now?

Reply:
we could add a type tag arg to the PTB like other txs, so when rosetta read back the TransactionData it will know it's a pay sui transfer.

Reply:
if there are no type tag arg (transactions created outside rosetta), we can parse it as a move call transaction without mistakenly parse into a pay sui tx